docs/ingress.md

```diff
@@ -88,6 +88,26 @@ nginx:
   httpsNodePort: 31001
 ```
 
+## Setting up NodePort using kubadm-dind-cluster
+
+First,  obtain the IP address of one of the two Kubernetes worker nodes.
+```shell
+kubectl describe node kube-node-2 | grep InternalIP
+```
```

Comment:
Why do you use `kube-node-2` here? I see in the `build-helm.sh` you are using `kube-node-2` to run invoker and using `kube-node-1` to get the IP address of `WSK_HOST`. Can you explain how to determine which node IP should be configured as `WSK_HOST`?

Reply:
Added some documentation. Good catch.  I meant to use kube-node-1 here.